src/tools/rbd_mirror/image_sync/ObjectCopyRequest.cc

```diff
@@ -114,6 +114,8 @@ void ObjectCopyRequest<I>::send_read_object() {
                << std::get<2>(sync_op) << dendl;
       op.sparse_read(std::get<1>(sync_op), std::get<2>(sync_op), &std::get<4>(sync_op),
                      &std::get<3>(sync_op), nullptr);
+      op.set_op_flags2(LIBRADOS_OP_FLAG_FADVISE_SEQUENTIAL |
```

Comment:
Super nit: the SEQUENTIAL flag doesn't really do anything since we operate on a whole object at a time. If we were performing small 512 - 4096 byte IOs in sequence, that would be the use-case for that flag. Not worth the effort to change, though.